### PR TITLE
Question Controller 2

### PR DESCRIPTION
**Pull request by JackyChiu**
#61 again

integrated with the frontend

but it does break the client app when you go to app/surveys after completeing a survey

### CODE REVIEW

File: src/main/java/com/example/survey/RestDataConfig.java [hunk 1]
```diff
@@ -1,16 +1,15 @@
...
 import org.springframework.data.rest.core.config.RepositoryRestConfiguration;
 import org.springframework.data.rest.webmvc.config.RepositoryRestConfigurerAdapter;
 import org.springframework.stereotype.Component;
+import org.w3c.dom.ranges.Range;
 
 @Component
 public class RestDataConfig extends RepositoryRestConfigurerAdapter {
     @Override
     public void configureRepositoryRestConfiguration(RepositoryRestConfiguration config) {
-        config.exposeIdsFor(Survey.class, Question.class, Answer.class);
```
Comment (bkzhang) [inline src/main/java/com/example/survey/RestDataConfig.java]: 👀

File: src/main/java/com/example/survey/RestDataConfig.java [hunk 2]
```diff
@@ -1,16 +1,15 @@
...
+        config.exposeIdsFor(Survey.class, Question.class, RangeQuestion.class, OptionQuestion.class, TextQuestion.class, Answer.class);
```
Comment (bkzhang) [inline src/main/java/com/example/survey/RestDataConfig.java]: 👀

**Comment by dipshit** [comment]
While reviewing this I had a lot of issues getting `yarn install` working, even after deleting `node_modules` and then the whole repo and recloning it. Can you lmk via imessage or something when you fix the merge conflicts I made so I can re-attempt a review? Hopefully that will fix whatever issue I had, because building from scratch did not

**Comment by JackyChiu** [comment]
I hate the commits on this, i'm going to reopen this on a new branch

**Pull request closed without merging**